_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,242 @@
+
+`Advanced settings` allows you to upload a JSON configuration.
+
+(Screenshot)
+
+#### Apply policy
+
+If you analyze time-series data, you likely prioritize new data over old data. You might periodically perform certain operations on older indices, such as reducing replica count or deleting them.
+
+[Index State Management]({{site.url}}{{site.baseurl}}/im-plugin/ism/index/) (ISM) is a plugin that lets you automate these periodic, administrative operations by triggering them based on changes in the index age, index size, or number of documents. The admin UI uses the ISM plugin to define policies that automatically handle index rollovers or deletions to fit your use case.
+
+For example, you can define a policy that moves your index into a `read_only` state after 30 days and then deletes it after a set period of 90 days. You can also set up the policy to send you a notification message when the index is deleted.
+
+You might want to perform an index rollover after a certain amount of time or run a `force_merge` operation on an index during off-peak hours to improve search performance during peak hours.
+
+To apply a policy, select the index you want to apply the policy to in the `Indices` list under `Index Management`. Then select the `Actions` button, and select `Apply policiy` from the drop-down menu.
+
+(Screenshot)
+
+#### Close
+
+The [close]({{site.url}}{{site.baseurl}}/api-reference/index-apis/close-index/) operation closes an index. Once an index is closed, you cannot add data to it or search for any data within the index.
+
+To close an index, select the index you want to close in the `Indices` list under `Index Management`. Then select the `Actions` button, and select `Close` from the drop-down menu.
+
+(Screenshot)
+
+#### Open
+
+The [open]({{site.url}}{{site.baseurl}}/api-reference/index-apis/open-index/) index operation opens a closed index, letting you add or search for data within the index.
+
+To open an index, select the index you want to open in the `Indices` list under `Index Management`. Then select the `Actions` button, and select `Open` from the drop-down menu.
+
+(Screenshot)
+
+#### Reindex
+
+The [reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/) operation lets you copy all or a subset of your data from a source index into a destination index.
+
+To reindex an index, select the index you want to reindex in the `Indices` list under `Index Management`. Then select the `Actions` button, and select `Reindex` from the drop-down menu.
+
+(Screenshot)
+
+#### Shrink
+
+The [shrink]({{site.url}}{{site.baseurl}}/api-reference/index-apis/shrink-index/) index operation moves all of your data in an existing index into a new index with fewer primary shards.
+
+To shrink an index, select the index you want to shrink in the `Indices` list under `Index Management`. Then select the `Actions` button, and select `Shrink` from the drop-down menu.
+
+(Screenshot)
```

Comment:
There is a minor change in the shrink index page lately, we have removed the warning `A copy of every shard must reside on the same node`, can you take a new screenshot for the shrink operation?

Reply:
will do